doc/source/reference/ndarray.rst

```diff
@@ -1,6 +1,7 @@
 -------------------
 N-Dimensional Array
 -------------------
+The multidimensional data array class. This class consists of a set of useful parameters and functions that allow not only to define an array correctly, but also to handle it in a simple way, being able to copy this array and transform it from a file or NumPy format to the ironArray format.
```

Comment:
Suggestion:

The multidimensional data array class. This consists of a series of parameters and functions that allow not only to construct an array, but also to handle it (for example, copying this to another array, either storing it into the native ironArray format file or converting it into a NumPy array).